pkg/operator/clustermembercontroller/clustermembercontroller.go

```diff
@@ -270,6 +279,41 @@ func (c *ClusterMemberController) sync() error {
 			return updateError
 		}
 		klog.Infof("All cluster members observed, scaling complete!")
+		if c.IsMember("etcd-bootstrap") {
+			c.eventRecorder.Event("BootstrapSafeToRemove", "scaling complete, bootstrap safe to remove")
+			_, _, updateErr := v1helpers.UpdateStatus(c.operatorConfigClient, v1helpers.UpdateConditionFn(
+				operatorv1.OperatorCondition{
+					Type:    ConditionBootstrapSafeToRemove,
+					Status:  operatorv1.ConditionTrue,
+					Reason:  "ScalingComplete",
+					Message: "cluster-etcd-operator has scaled, bootstrap safe to remove",
+				}))
+			if updateErr != nil {
+				klog.Errorf("clustermembercontroller:sync: error updating status: %#v", updateErr)
+				return updateErr
+			}
+		} else {
+			c.eventRecorder.Event("BootstrapRemoved", "scaling complete, bootstrap was not found")
+			_, _, updateErr := v1helpers.UpdateStatus(c.operatorConfigClient,
+				v1helpers.UpdateConditionFn(
+					operatorv1.OperatorCondition{
+						Type:    ConditionBootstrapSafeToRemove,
+						Status:  operatorv1.ConditionTrue,
+						Reason:  "ScalingComplete",
+						Message: "cluster-etcd-operator has scaled, bootstrap safe to remove",
+					}),
+				v1helpers.UpdateConditionFn(
+					operatorv1.OperatorCondition{
+						Type:    ConditionBootstrapRemoved,
```

Comment:
this condition is set by the bootstrap removed controller, not this code. delete this